nilearn/datasets.py

```diff
@@ -449,6 +445,8 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
     # Determine filename using URL
     parse = _urllib.parse.urlparse(url)
     file_name = os.path.basename(parse.path)
+    if file_name == '':
```

Comment:
I am perhaps wrong on this one, but I thought that checking whether a string is empty is most elegantly done by ```not``` operator.